app/Http/Requests/SaveUserRequest.php

```diff
@@ -35,7 +35,10 @@ public function rules()
             {
                 $rules['first_name'] = 'required|string|min:1';
                 $rules['username'] = 'required_unless:ldap_import,1|string|min:1';
-                $rules['password'] = Setting::passwordComplexityRulesSaving('store');
+                if ($this->request->get('ldap_import') == false)
```

Comment:
I'm not sure this works here. If the user was imported via LDAP, that shouldn't mean that admin-imposed password restrictions should be bypassed forever. Sometimes it's a one-time import, not an ongoing sync, and what happens if LDAP password requirements change? At that point you'd have an insecure password for an older user, and they'd never be flagged since they bypass the Snipe-IT password requirements, (which would in theory be modified up with the LDAP ones).

Reply:
The current LDAP sync doesn't check against it from what i can see which is why I implemented as the password is randomly generated (Which I also used the same random generate code), I couldn't also get around the required without removing this. 

Can you think of a better way to doing this?

Reply:
From what I can see it also doesn't look like the LDAP once are every updates with the sync anyway?  Only when the user logs in and has Sync password selected but this also doesnt seem to go through the password complexity rules?

Reply:
Any update on this one @snipe :) thanks!

Reply:
@snipe Is it possible to merge this Feature?
See #5328

Reply:
@snazy2000 could you share your powershell script?